### DISCUSSION TITLE
How to edit node information directly from code

### DISCUSSION THREAD

**Discussion by jiaxu0715** [SSShooter/mind-elixir-core#133] [Q&A]
Hello! I have been wondering how I could edit the text information in nodes directly through editing the website source, since currently there only seem to be options for moving, inserting/removing nodes. Is there a location in the source code of the app where the data for nodes (maybe with the "Data Structure" section of the documentation) is stored? Thank you!

**Comment by SSShooter** [comment]
modify `instance.nodeData` then `instance.refresh()`
